src/FlashList.tsx

```diff
@@ -136,6 +138,19 @@ class FlashList<T> extends React.PureComponent<
     if (Number(this.props.numColumns) > 1 && this.props.horizontal) {
       throw new CustomError(ExceptionList.columnsWhileHorizontalNotSupported);
     }
+    if (
+      this.props.horizontal &&
+      this.props.experimentalScrollPositionManagement
+    ) {
+      throw new CustomError(ExceptionList.horizontalMaintainScrollNotSupported);
+    }
+
+    if (
+      this.props.experimentalScrollPositionManagement &&
+      this.props.renderScrollComponent
+    ) {
+      throw new CustomError(ExceptionList.customMaintainScrollNotSupported);
```

Comment:
I'm wondering if people can still use custom scroll component by using DoubleSidedScrollView instead of ScrollView in their component? In that case we can export DoubleSidedScrollView from FlashList too.